azure-kusto-data/source/client.ts

```diff
@@ -157,6 +188,16 @@ export class KustoClient {
         return this._doRequest(endpoint, executionType, headers, payloadContent, timeout, properties);
     }
 
+    private getDb(db: string | null) {
+        if (db == null) {
+            if (this.defaultDatabase == null) {
```

Comment:
In c# initalCatalog default is "NetDefaultDB"

Reply:
Interesting, do we want this here too?
In c# it's set on the KustoConnectionStringBuilder level

Reply:
Done